cypress/integration/constrain-creation.feature

```diff
@@ -6,7 +6,12 @@ Feature: create constraints by selecting nodes from the tree
     When I use public study 'CATEGORICAL_VALUES ' as a constraint
     Then there are '3' subjects
 
```

Comment:
looks like the time to correct the file name: constrain-creation -> constraint-creation :)

Reply:
ah, didn't even notice that ;) I will rename it!